lib/tests/2wp.js

```diff
@@ -139,6 +207,34 @@ const assert2wpBalancesAfterSuccessfulPegin = async (initial2wpBalances, peginVa
 
 };
 
+const addInputs = (tx, senderUtxosInfo) => {
+  senderUtxosInfo.utxos.forEach(utxo => {
+    tx.addInput(Buffer.from(utxo.txid, 'hex').reverse(), utxo.vout);
+  });
+};
+
+const addChangeOutputs = (tx, changeAddress, changeInSatoshis) => {
```

Comment:
I think it should say `addChangeOutput` because it is just adding a single output. Wdyt?

Reply:
Thanks. I updated it in the next pr. https://github.com/rsksmart/rootstock-integration-tests/pull/94

Reply:
Nice